CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [7.5.1] - 2024-04-??
 
+- Added: The Discord bot now mentions the game when describing a preset.
 - Changed: The output after verifying the installation has been made less misleading.
 - Nothing yet. Space reserved in case needed.
```

Comment:
Someone forgot to remove that in a PR

Reply:
the whole changelog thing will need to be changed in the future anyway, since the next release is 8.0.0. So I'd only then would like to remove 7.5.1 and 7.6.0 and their placeholder text.

Reply:
That doesn't sound like a reson to keep a placeholder if it already contains entries.